bubblesub/cmd/file_properties.py

```diff
@@ -318,7 +316,7 @@ def _load(self) -> None:
             int(T.cast(str, self._api.subs.meta.get("PlayResX", "0"))),
             int(T.cast(str, self._api.subs.meta.get("PlayResY", "0"))),
         )
-        if (
+        if (  # pylint: disable=too-many-boolean-expressions
```

Comment:
@rr- 

Can we reduce these many boolean expressions?

Reply:
Would the following work?
```diff
         if (
             old_res != new_res
-            and old_res[0]
-            and old_res[1]
-            and new_res[0]
-            and new_res[1]
+            and all(old_res)
+            and all(new_res)
             and await show_prompt(
                 "The resolution was changed. "
                 "Do you want to rescale all the styles now?",
                 self,
             )
         ):
```

Reply:
Yep, fixed. Thanks! :)